[2024-11-30T09:05:45.075+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_etl.load manual__2024-11-30T09:05:18.307060+00:00 [queued]>
[2024-11-30T09:05:45.098+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_etl.load manual__2024-11-30T09:05:18.307060+00:00 [queued]>
[2024-11-30T09:05:45.100+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-11-30T09:05:45.145+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): load> on 2024-11-30 09:05:18.307060+00:00
[2024-11-30T09:05:45.164+0000] {standard_task_runner.py:60} INFO - Started process 929 to run task
[2024-11-30T09:05:45.172+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_etl', 'load', 'manual__2024-11-30T09:05:18.307060+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmp8gh7e959']
[2024-11-30T09:05:45.178+0000] {standard_task_runner.py:88} INFO - Job 9: Subtask load
[2024-11-30T09:05:45.300+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_etl.load manual__2024-11-30T09:05:18.307060+00:00 [running]> on host 8d09c1752432
[2024-11-30T09:05:46.193+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-11-30T09:05:18.307060+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-30T09:05:18.307060+00:00'
[2024-11-30T09:05:46.825+0000] {logging_mixin.py:188} INFO - Data saved to ./data/weather_data.csv
[2024-11-30T09:05:46.828+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-11-30T09:05:46.884+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_etl, task_id=load, execution_date=20241130T090518, start_date=20241130T090545, end_date=20241130T090546
[2024-11-30T09:05:47.085+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-30T09:05:47.250+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
